Formula/less.rb

```diff
@@ -3,6 +3,7 @@ class Less < Formula
   homepage "http://www.greenwoodsoftware.com/less/index.html"
   url "http://www.greenwoodsoftware.com/less/less-551.tar.gz"
   sha256 "ff165275859381a63f19135a8f1f6c5a194d53ec3187f94121ecd8ef0795fe3d"
+  license "GPL-3.0-or-later OR BSD-2-Clause"
```

Comment:
IANAL, but:
1. the phrase "Less License" at the top may actually indicate that "this is the license that applies to the Less software product", rather than "this series of words shall be distinguished from all other identical series of words as the *Less License*"
1. in a legal context, the actual words of the license matter far more than the name attached to said words, so the [duck test](https://en.wikipedia.org/wiki/Duck_test) should apply here...especially if the author's license summary and the actual license text say different things.

Therefore, I think `BSD-2-Clause` is perfectly fine.

Reply:
Well, to be perfectly accurate, the license in [less's LICENSE file](https://github.com/gwsw/less/blob/master/LICENSE) is not 100% identical to the [BSD-2-Clause](https://spdx.org/licenses/BSD-2-Clause.html). In the second clause, it changes from

> 2. Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the following disclaimer in the documentation and/or other materials provided with the distribution. 

to simply

> 2. Redistributions in binary form must reproduce the above copyright notice in the documentation and/or other materials provided with the distribution.

Notice how "above copyright notice, this list of conditions and the following disclaimer" was changed to only "the above copyright notice".

None of the SPDX-encoded BSD licenses seem to match this phrasing exactly, but indeed the standard BSD-2-Clause is the closest match. Not sure if that slight deviation means we should treat it as a different license.